doc/source/parameters/dem/insertion_info.rst

```diff
@@ -0,0 +1,41 @@
+Insertion Info
+-------------------
+Particle insertion information is defined in this section. This information includes ``insertion method``, ``inserted number of particles at each insertion step``, ``insertion frequency``, dimensions of the insertion box, approximate initial distance between particles at insertion steps, and information about the randomness of initial positions of particles. If the insertion method is ``uniform``, the particles are uniformly (without randomness in initial location) inserted. On the other hand, in ``non_uniform`` insertion, initial locations of particles in `x` and `y` directions are chosen randomly. Using ``non_uniform`` insertion, insertion ``random number range`` and ``insertion random number seed`` are also required in the parameter handler file.
```

Comment:
Try to add here a more general explanation of how particle insertion works. You can add the explanation of the parameters one by one in a list format below.

Reply:
Explained. Plus, all the parameters are explained one by one in a list.